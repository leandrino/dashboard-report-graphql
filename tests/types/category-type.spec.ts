import CategoryType from '../../src/graphql/types/category-type'
// import { graphQLObjectType, graphQLString } from 'graphql'

describe('Category Type', () => {
  it('should to match snapshot', () => {
    expect(CategoryType).toMatchSnapshot()
  })
})
