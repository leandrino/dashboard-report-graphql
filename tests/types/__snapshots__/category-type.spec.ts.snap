// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category Type should to match snapshot 1`] = `"Category"`;
